.github/workflows/draft-release.yml

```diff
@@ -73,17 +73,25 @@ jobs:
                 let urlMap = new Map();
                 tuples.forEach(function(tuple) {
                   if (tuple[0].includes("linux")) {
-                    urlMap.set("linux", tuple[1])
+                    if (tuple[0].includes("sha256")) {
```

Comment:
I think we need to additionally check the architecture in the line above (see https://github.com/buildpacks/lifecycle/issues/682).

Reply:
Do you want to merge https://github.com/buildpacks/lifecycle/pull/680/files before?

Reply:
I don't think we could merge that one before we have a machine to test it. But we could just copy the fix over from there. Anyway this PR introduces a new problem in that the sha file could also be amd64 or arm64.

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/679/commits/b02eff553070a649a8f38a8bebb1e1336dbb43be.
I'm keeping this conversation open since I'm not sure that I've made all of the changes that you meant.
Please let me know what do you think. Thanks!